openreview/venue_request/process/revisionProcess.py

```diff
@@ -229,6 +229,12 @@ def process(client, note, invitation):
                 'required': False
             }
 
+            content['hide_fields'] = {
```

Comment:
@celestemartinez do you think the Revision invitation should have this field "hide_fields"? 

they can use the post submission stage, right? or this stage is expired after the decisions are posted?

Another option is to show the field in the Post Decision Stage form.

Reply:
The Post Submisison Stage has no expiration date, so this invitation can be used at any point, but I am not sure if every venue uses it. It might not be a bad idea to also include it in the Post Decision Stage form